provisionerd/proto/provisionerd.proto

```diff
@@ -109,6 +109,9 @@ message UpdateJobRequest {
     repeated provisioner.TemplateVariable template_variables = 4;
 	repeated provisioner.VariableValue user_variable_values = 5;
     bytes readme = 6;
+	// Heartbeat is true when the request is a heartbeat indicating job
+	// liveness.
+	bool heartbeat = 7;
```

Comment:
Hmm, why does this need this?

Reply:
I guess it doesn't really since I only use it in the update function